science/nds2-client-swig/Portfile

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem        1.0
+PortGroup         cmake 1.0
+
+name              nds2-client-swig
+version           0.16.3
+revision          0
+categories        science
+platforms         darwin
+license           GPL-2
+maintainers       ligo.org:ed.maros openmaintainer
```

Comment:
Add your GitHub handle here so you automatically get notified when someone submit a PR against this port.